echopop/live/live_acoustics.py

```diff
@@ -0,0 +1,337 @@
+    return pd.DataFrame(nasc_dict, index=[0])
+
+    # return pd.DataFrame([nasc_dict])
+
+
+def compute_nasc(
+    acoustic_data_df: pd.DataFrame, file_configuration: dict, echometrics: bool = True
+):
+
+    # Get spatial definitions, if any
+    # spatial_column = file_configuration["spatial_column"]
+
+    # Get stratum column, if any
+    gridding_column = file_configuration["gridding_column"]
+
+    # Integrate NASC (and compute the echometrics, if necessary)
+    # ---- Get number of unique sources
+    # if len(np.unique(acoustic_data_df["ping_time"])) > 1:
+    #     nasc_data_df = (
+    #         acoustic_data_df
+    #         .groupby(["longitude", "latitude", "ping_time", "source"] + spatial_column,
+    #                  observed=False)
+    #         .apply(integrate_nasc, echometrics, include_groups=False).unstack()
+    #         .reset_index()
+    #         .sort_values("ping_time")
+    #     )
+    # else:
+    nasc_data_df = (
+        acoustic_data_df.groupby(
+            ["ship_id", "longitude", "latitude", "ping_time", "source"] + gridding_column,
+            observed=False,
+        )
+        .apply(integrate_nasc, echometrics, include_groups=False)
+        .droplevel(-1)
+        .reset_index()
+        .sort_values("ping_time")
+    )
+    # ---- Amend the dtypes if echometrics were computed
+    if echometrics:
+        # ---- Set dtypes
+        nasc_data_df = nasc_data_df.astype(
+            {
+                "n_layers": int,
+                "mean_Sv": float,
+                "max_Sv": float,
+                "nasc_db": float,
+                "center_of_mass": float,
+                "dispersion": float,
+                "evenness": float,
+                "aggregation_index": float,
+                "occupied_area": float,
+            }
+        )
+        # ---- Reorder columns
+        nasc_data_df = nasc_data_df[
+            gridding_column
+            + [
+                "ship_id",
+                "longitude",
+                "latitude",
+                "ping_time",
+                "source",
+                "nasc",
+                "n_layers",
+                "nasc_db",
+                "mean_Sv",
+                "max_Sv",
+                "aggregation_index",
+                "center_of_mass",
+                "dispersion",
+                "evenness",
+                "occupied_area",
+            ]
+        ]
+
+    # Return the output
+    return nasc_data_df
+
+
+def format_acoustic_dataset(nasc_data_df: pd.DataFrame, file_configuration: dict, meta_dict: dict):
+
+    # Get acoustic database filename
+    acoustic_db = file_configuration["database"]["acoustics"]
+
+    # Create a copy of the dataframe
+    df = nasc_data_df.copy()
+
+    # Add population-specific columns (specified in the file configuration)
+    # TODO: Add to `yaml` file for configuration; hard-code for now
+    add_columns = ["number_density", "biomass_density"]
+    # ----
+    df[add_columns] = 0.0
```

Comment:
Just trying to make sure that I understand - these two columns are not supposed to be added here, and they are here as a temporary hardcoded fix?